src/app/clusters/electrical-energy-measurement-server/BUILD.gn

```diff
@@ -11,5 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-group("electrical-energy-measurement-server") {
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+config("electrical-energy-measurement-server-config") {
+  include_dirs = [ "." ]
```

Comment:
why is this needed? can we use paths out of `src` instead?

Reply:
Includes should be accessible by depending on the target, because they inherit https://github.com/project-chip/connectedhomeip/blob/master/src/BUILD.gn#L29 `src:includes` as a configuration (if they do not we can add it, however I belive this is a dependency on all our base libraries like lib support/core and such)

So you will have to `#include <app/clusters/electrical-energy-measurement-server/...>` instead of `include "..."`, however I believe that makes things more consistent.